src/App.tsx

```diff
@@ -21,11 +54,14 @@ export const App: React.FC = () => {
             data-cy="perPageSelector"
             id="perPageSelector"
             className="form-control"
+            defaultValue={ITEMS_PER_PAGE_INITIAL}
+            onChange={e => handerItemsPerPage(e)}
           >
             <option value="3">3</option>
```

Comment:
Would be great to create an array with possible options and iterate over it, to not repeat the same code for each option